### ISSUE TITLE
Question on Shift Module

### ISSUE THREAD

**Issue by doubbblek** [Phoenix1327/tea-action-recognition#10]
First of all, thanks for proposing and sharing such great work! 

Now I saw that when you implement `ShiftModule` in [tea50_8f.py](https://github.com/Phoenix1327/tea-action-recognition/blob/f59e2f8166dfe78ca06d4ebc7d456665a3c100d9/ops/tea50_8f.py#L93), you first reshape the tensor and then apply a 1D Convolution.

I wonder if such shift convolution is equivalent to:
1) reshape tensor of shape (batch*t, c, h, w) to (batch, c, t, h, w) 
2) Apply `nn.Conv3d(in_channel=c, out_channel=c, kernel_size=(3, 1, 1), stride=(1, 1, 1), padding=(1, 0, 0), groups=c)`.

I have read your paper and kept wondering this problem. Again, thanks for your great work.

Best Regards,
Kade
